examples/alert_util.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright (2012-2019) Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+###
+import cmd
+import csv
+import os
+from datetime import datetime
+from collections import Counter
+from hpOneView.oneview_client import OneViewClient
+from config_loader import try_load_from_file
+
+
+CONFIG = {}
+
+
+class OneView(object):
+    """
+    OneView methods
+    """
+
+    def __init__(self):
+        self.client = self.connect(CONFIG)
+
+    def connect(self, config):
+        # Try load config from a file (if there is a config file)
+        config = try_load_from_file(config)
+        return OneViewClient(config)
+
+    def get_alerts(self, _filter='', _view=''):
+        print("\nFilter: {}\n".format(_filter))
+        resp = self.client.alerts.get('/rest/alerts?filter={}'.format(_filter))
```

Comment:
Do we need two get calls in this method? 
Can we use one get_all method with the filter to get all the alerts then print the total count and return the result?

Reply:
Ok, got it.